components/collector/ci/pip-install.sh

```diff
@@ -10,8 +10,5 @@ run () {
 }
 
 # Install the requirements
-run pip install --ignore-installed --quiet -r requirements/requirements-base.txt
-run pip install --ignore-installed --quiet -r requirements/requirements.txt
```

Comment:
We don't need these two anymore because they are now included within the requirements-dev, right? But where does the content of the the requirements-internal come from then?